api/automoderation.go

```diff
@@ -0,0 +1,72 @@
+package api
+
+import (
+	"github.com/diamondburned/arikawa/v3/discord"
+	"github.com/diamondburned/arikawa/v3/utils/httputil"
+)
+
+// Get a list of all rules currently configured for the guild. Returns a list of auto moderation rule objects for the given guild.
```

Comment:
Please edit these comments to be in the format `// ListAutoModerationRules gets a list of`...

Reply:
Updated in [5ac59ad](https://github.com/diamondburned/arikawa/pull/444/commits/5ac59ad300e1c113bfa99d7cf868b05079647505)